### PR TITLE
RK-9361 - Add min resources to dop

### PR DESCRIPTION
**Pull request by adilrookout**
Resources were defined in the values.yaml but not used in the deployment.yaml until this PR

### CODE REVIEW

**Comment by Gilwe** [comment]
Did you `helm template` it locally? I just wanna be sure about the indent

**Comment by adilrookout** [comment]
Sure :

image

File: charts/datastore/templates/deployment.yaml [hunk 1]
```diff
@@ -50,6 +50,8 @@ spec:
         - name: {{ template "datastore.name" . }}
           image: "{{ .Values.image.repository }}:{{ default "latest" .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
```
Comment (adilrookout) [inline charts/datastore/templates/deployment.yaml:54]: indent copied from controller chart

Comment (adilrookout) [reply charts/datastore/templates/deployment.yaml:54]: @Gilwe ☝️

**Review by Gilwe — APPROVED** [review]

**Pull request merged**